x-pack/plugins/ml/public/application/datavisualizer/datavisualizer_selector.tsx

```diff
@@ -102,7 +102,7 @@ export const DatavisualizerSelector: FC = () => {
               title={
                 <FormattedMessage
                   id="xpack.ml.datavisualizer.selector.importDataTitle"
-                  defaultMessage="Import data"
+                  defaultMessage="Visualize data from file"
```

Comment:
```suggestion
                  defaultMessage="Visualize data from a file"
```

Reply:
Done in https://github.com/elastic/kibana/pull/123019/commits/5e903277b71de4ea1b8dbea10d25dccfe1d6a83b